mongodb_task2_completed.txt

```diff
@@ -0,0 +1,201 @@
+1.	How many records does each airline class have? Use $project to show result as { class: "Z", total: 999 }
+
+db.airlines.aggregate([
+  {
+    $group: {
+      _id: '$class',
+      total: { $sum: 1 }
+    }
+  },
+  {
+    $project: {
+      _id: 0,
+      class: '$_id',
+      total: '$total'
+    }
+  }
+]);
+
+{ "class" : "F", "total" : 140343 }
+{ "class" : "L", "total" : 23123 }
+{ "class" : "P", "total" : 5683 }
+{ "class" : "G", "total" : 17499 }
+
+
+2.  What are the top 3 destination cities outside of the United States (destCountry field, not included) with the highest average passengers count? Show result as { "avgPassengers" : 2312.380, "city" : "Minsk, Belarus" }
+
+db.airlines.aggregate([
+  {
+    $match: {
+      destCountry: { $ne: 'United States' }
+    }
+  },
+  {
+    $group: {
+      _id: '$destCity',
+      avgPassangers: { $avg: '$passengers' }
+    }
+  },
+  {
+    $sort: { avgPassangers: -1 }
+  },
+  {
+    $limit: 3
+  },
+  {
+    $project: {
+      _id: 0,
+      city: '$_id',
```

Comment:
`city: "$_id.city"` if you agree with line 35 (to add `$destCountry` as a part of _id)